client/src/app/[locale]/(login)/sign-up/sign-up.tsx

```diff
@@ -4,27 +4,32 @@ import { useTranslations } from 'next-intl';
 import { FieldArray, Form, Formik, FormikValues } from 'formik';
 
 import {
-  Title,
-  SmallBtn,
   Button,
+  CheckboxRadioField,
   DateField,
   FileField,
   InputField,
-  CheckboxRadioField,
-  organizationValidation,
   organizationInitialValues,
+  organizationValidation,
+  SmallBtn,
+  Title,
```

Comment:
sort by length

Reply:
All imports are formatted and optimized via WebStorm.
If there should be other formatted please add it to the project.